source/includes/administration/_knowledge_base.md

```diff
@@ -118,6 +132,9 @@ Attributes | &nbsp;
 `categories.createdAt`<br/>*string* | The creation date.
 `categories.updatedAt`<br/>*string* | The update date.
 `categories.featured`<br/>*boolean* | If the category needs to be displayed.
+`categories.externalDocumentation`<br/> | List of URL and language object. Only one of articles or external documentation is provided for a category (example above shows both only as an example for the object structure).
```

Comment:
List of URL and language objects. Only either 1) articles, or, 2) external documentation, will be provided for a category (example above shows both only as an example for the object structure).

Reply:
53c936aeb20dd7b658b0cde164b7849039f83219